zboss/CHANGELOG.rst

```diff
@@ -9,22 +9,22 @@ Changelog
 
 All notable changes to this project in |NCS| are documented in this file.
 
-Master branch
-*************
+nRF Connect SDK v1.5.0
+**********************
 
 Changes
 =======
 
-* Updated ZBOSS stack to version ``3.3.0.6+11_30_2020`` that includes the Cortex-M33 CPU libraries.
-  For detailed release notes for this stack version, see `ZBOSS stack release notes`_ for the Master branch.
+Updated ZBOSS stack to version ``3.3.0.6+11_30_2020`` that includes the Cortex-M33 CPU libraries.
+For detailed release notes for this stack version, see `ZBOSS stack release notes`_ for the Master branch.
```

Comment:
Master branch was renamed, so it will be renamed on that page as well. Use "nRF Connect SDK v1.5.0" here.